README.md

```diff
@@ -19,6 +19,7 @@ Usage:	/entrypoint.sh COMMAND
 Pentaho Data Integration (PDI)
 
 Options:
+  encr password		Encrypt a password
```

Comment:
This should say "obfuscate" rather than "encrypt" (here and in the help text).

Reply:
done